src/views/notification/index.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+use yii\helpers\Html;
+use kartik\grid\GridView;
+Use kartik\editable\Editable;
+use webzop\notifications\model\NotificationType;
+use kartik\daterange\DateRangePicker;
+
+/* @var $this yii\web\View */
+/* @var $searchModel webzop\notifications\model\NotificationSearch */
+/* @var $dataProvider yii\data\ActiveDataProvider */
+
+
+$this->title = 'Notifications';
+$this->params['breadcrumbs'][] = $this->title;
+?>
+<div class="notifications-index">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js" crossorigin="anonymous"></script>
+    <h1><?= Html::encode($this->title) ?></h1>
+
+    <?php // echo $this->render('_search', ['model' => $searchModel]); ?>
+
+    <?= GridView::widget([
+        'dataProvider' => $dataProvider,
+        'filterModel' => $searchModel,
+        'formatter' => ['class' => 'yii\i18n\Formatter','nullDisplay' => ''],
+        'columns' => [
+            'type' => [
+                'attribute' => 'type',
+                'filter' => NotificationType::find()->select('name')->indexBy('id')->column(),
+                'filterInputOptions' => [
+                    'class' => 'form-control',         
+                    'prompt' => 'All'
+                ],
+                'value' => function ($model, $key, $index, $column){
+                    return $model->notificationsType->name;
+                },
+                'contentOptions' => function ($model, $key, $index, $column) {
+                    return ['style' => 'background-color:'.($model->notificationsType->color)];
+                },               
+                'vAlign' => 'middle',
+                'hAlign' => 'center',
+                'width'=>'170px',
+            ],
+            'message' => [
+                'attribute' => 'message',
+                'hAlign' => 'center',
+                'width'=>'200px',
+            ],
+            'created_at' => [
+                'attribute' => 'created_at',
+                'label' => 'Datetime',
+                'format' => 'datetime',
+                'filter' => DateRangePicker::widget([
+                    'model' => $searchModel,
+                    'attribute' => 'created_at_filter',
+                    'convertFormat' => true,
+                    'pluginOptions' => [
+                        'timePicker'=>true, 
+                        'timePickerIncrement'=>10,
+                        'locale' => ['format' => 'd-m-Y g:i A']
+                    ]
+                ]),
+                'vAlign' => 'middle',
+                'hAlign' => 'center',
+            ],
+
+            'timeago' => [
+                'attribute' => 'timeAgo',
+                'value' => function ($model, $key, $index, $column){
+                    return $model->timeAgo;
```

Comment:
this is not needed, by specifying the attribute 'timeAgo' it automatically returns ` $model->timeAgo`